src/composables/useDebounce.ts

```diff
@@ -0,0 +1,9 @@
+export function useDebounce() {
+  let timeout: any = null;
+  return function (fnc, delayMs) {
+    clearTimeout(timeout);
+    timeout = setTimeout(() => {
+      fnc();
+    }, delayMs || 500);
```

Comment:
Is it good to reduce this delay? 🤔 on mouse leave, it feels like it is taking some time for popper to go away.